src/fidgets/ui/LinksInput.tsx

```diff
@@ -0,0 +1,282 @@
+import React, { forwardRef, useCallback, useState, useEffect } from "react";
+import { Link } from "./Links";
+import { FaPlus, FaImage, FaLink } from "react-icons/fa";
+import {
+  Avatar,
+  AvatarFallback,
+  AvatarImage,
+} from "@/common/components/atoms/avatar";
+import styled from "styled-components";
+import { FaPencil } from "react-icons/fa6";
+import CSSInput from "@/common/components/molecules/CSSInput";
+
+export interface LinksInputProps {
+  value: Link[];
+  onChange?: (value: Link[]) => void;
+}
+
+const Box = styled.div`
+  width: 100%;
+`;
+
+const TextFieldRoot = styled.div<{ isActive: boolean }>`
+  display: flex;
+  align-items: center;
+  position: relative;
+  background: white;
+  border: ${({ isActive }) =>
+    isActive ? "2px solid lightblue" : "1px solid lightgray"};
+  padding: 5px;
+  border-radius: 4px;
+  width: 100%;
+`;
+
+const TextFieldSlot = styled.div`
+  display: flex;
+  align-items: center;
+`;
+
+const TextFieldInput = styled.input`
+  border: none;
+  outline: none;
+  flex: 1;
+  padding: 0 5px;
+`;
+
+const LinkContainer = styled.div`
+  margin-bottom: 1em;
+`;
+
+const Title = styled.p`
+  font-size: 0.875rem;
+  color: black;
+  font-weight: bold;
+`;
+
+const SubTitle = styled.p`
+  font-size: 0.875rem;
+  color: gray;
+`;
+
+const AddLinkButton = styled.button`
+  display: flex;
+  align-items: center;
+  margin-top: 10px;
+  background: none;
+  border: none;
+  color: #007bff;
+  cursor: pointer;
+  font-size: 1rem;
+
+  &:hover {
+    text-decoration: underline;
+  }
+`;
+
+const LinksInput = forwardRef<HTMLInputElement, LinksInputProps>(
+  (props, ref) => {
+    const { value = [], onChange } = props;
+
+    const [linkStates, setLinkStates] = useState<
+      {
+        isAvatarInputDisplayed: boolean;
+        confirmed: boolean;
+        isActive: boolean;
+      }[]
+    >([]);
+
+    useEffect(() => {
+      if (value.length === 0) {
+        const defaultLink = {
+          text: "My Awesome Link",
+          url: "https://nounspace.com",
+          avatar:
+            "https://nounspace.mypinata.cloud/ipfs/QmSkE84i5LoA6b6VZBZe4Ku4MTo5ueYhCc3Hh7EY8JM3pb",
```

Comment:
will also put this as a constant

Reply:
added to public folder, we probably gonna user noggles logo again (changing the default link to Nouns one as requested by willy)